solr/config/schema.xml

```diff
@@ -15,7 +15,7 @@
     <field name="lng" type="tdouble" stored="true" indexed="true" multiValued="false"/>
     <field name="original_pid_tesim" type="pid_text" stored="true" indexed="true" multiValued="true"/>
 
-    <field name="full_title_ng" type="text_en_ng" stored="false" indexed="true" multiValued="true"/>
+    <field name="cho_title_ng" type="text_en_ng" stored="false" indexed="true" multiValued="true"/>
```

Comment:
I don't /think/ so. This is just an ngrams field to support autocomplete in sir trevor widgets, so a little slop here and there oughtn't make a big difference.